### ISSUE TITLE
Default `slider` widget is invisible and does not respond to events

### ISSUE THREAD

**Issue by sudormrfbin** [lapce/floem#278]
The slider widget is invisible by default:

```rust
use floem::reactive::create_rw_signal;
use floem::view::View;
use floem::views::Decorators;
use floem::widgets::slider;

pub fn bar() -> impl View {
    let percent = create_rw_signal(0.5);

    slider::slider(move || percent.get())
        .on_change_pct(move |p| percent.set(p))
        // .style(|s| s.height(15).width(200))
}

pub fn main() {
    floem::launch(bar);
}
```

The height and width has to be set manually for it to be displayed (uncomment the `style()` call). Moreover, the slider doesn't respond to any event (mouse clicks, arrows, etc) and the initial position of the slider handle also seems to be set incorrectly (the signal is initialized to 50% but the handle is at 0%):

![Screenshot_2024-01-19_22-03-35](https://github.com/lapce/floem/assets/23398472/edfd54e2-0ffc-4d72-9af3-a3c60700c08b)

To reiterate, the issues are:
- Invisible without styling
- Doesn't respond to mouse clicks
- Incorrect initial value

#### Floem Version
ae59e6ce

#### Operating System
Linux (Xorg)

**Comment by jrmoulton** [comment]
This might be an issue with some changes I made. I'll take a look

**Comment by jrmoulton** [comment]
Thanks for the detailed report. All of the issues should be fixed (and more) once #279 gets merged
